[S96] add tile and geojson layers to layer manager

gridcell opened the pull request.

gridcell committed:
add tile and geojson layers to layer manager

gridcell committed:
add tile layer subdomains

gridcell committed:
update api urls for prod

gridcell commented:
![](https://github.trello.services/images/mini-trello-icon.png) [api/map layer mappings](https://trello.com/c/OoZh5Ddy/96-api-map-layer-mappings)

saanobhaai commented on src/lib/maplayers.js:
```diff
@@ -1,33 +1,82 @@
+          Name: ${name}
+        </div>
+        <div>
+          Total Area: ${area}km<sup>2</sup>
```
To avoid potential confusion, let's say `Total country area`

saanobhaai commented on src/lib/maplayers.js:
```diff
@@ -1,33 +1,82 @@
+      return msg
+    })
+
+    this.api = new SecureApi()
```
Does this need to be redeclared?

saanobhaai commented on src/lib/maplayers.js:
```diff
@@ -1,33 +1,82 @@
+  }
+
+  getTclLayer(country) {
+    const date = '2006-01-01'
```
Could we maybe make date also a constructor parameter, and temporarily set both it and country as `let`s at the top of `leafletMap.index.js`? That way it will be easier to change them from other parts of the app down the road, when we have a date slider and country picker.

saanobhaai reviewed (changes requested):
Awesome as always, just wondering about a couple of small things that might make life easier for our future selves

gridcell committed:
CR changes

- move TCL layer args to leaflet component
- remove extra SecureApi
- Change label for area in map popup

gridcell committed:
remove using addLayer for base layer
